### DISCUSSION TITLE
Tailscale exit node adblocking

### DISCUSSION THREAD

**Discussion by Hootie81** [hassio-addons/addon-adguard-home#422] [General]
I'm using this AdGuard addon with Tail Scale addon both within HAOS, i would like to utilize HAOS as an exit node and also provide ad blocking to the device using the exit node. currently AdGuard doesnt listen to the tailnet IP. 

While Adguard does listen on local host 127.0.0.1 and i can set the HA tailscale IP as the nameserver on tailscale DNS settings and have override local DNS set, it appears that ad blocking does not happen. i assume that it uses the HA DNS settings from the network adaptor settings. 

is it possible to get the tail scale IP of HA as a listening address? this way i can point the traffic to adguard and not the network adaptor.. or am i missing something?

my current workaround is to use a windows machine to act as an exit node and point it to the adguard DNS. however i dont want to have this machine on 24/7, or another device on the network just to perform this function. i can see several forum posts asking the same question but cant see a clear working solution.

**Comment by Permanently** [comment]
Try exposing specifically the LAN IP of the HA AdGuard machine, then setting that as the DNS IP.
